guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<blockquote>The inference-only classes in<code>arcgis.learn.text</code> submodule is based on <a href="https://huggingface.co/transformers/v3.3.0/index.html" target="_blank">Hugging Face Transformers</a> library. T</blockquote><p> Suggestion: The pretrained models/inference-only models available in <code>arcgis.learn.text</code> submodule <s>is</s> are based on <a href="https://huggingface.co/transformers/v3.3.0/index.html" target="_blank">Hugging Face Transformers</a> library. </p><p> </p><blockquote> <a href="https://huggingface.co/transformers/v3.0.2/pretrained_models.html" target="_blank">This page</a> mentions different transformers architectures <a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/files/#References" target="_blank">[2]</a> which come in different sizes (model parameters), trained on different languages /corpus, having different attention heads, etc.</blockquote><blockquote><br></blockquote><p>Suggestion: <a href="https://huggingface.co/transformers/v3.0.2/pretrained_models.html" target="_blank">This page</a> mentions different <s>transformers</s> transformer architectures <a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/files/#References" target="_blank">[2]</a> which come in different sizes (model parameters), trained on different langu<strong>ages<s> </s>/corp</strong>us, having different attention heads, etc.</p><p> </p><p>langu<strong>ages /corp</strong>us --&gt; Not sure if there is any space before slash, if so need to remove</p><p> </p><blockquote>The usage of these models differs from rest of the <code>imagery</code> and <code>text</code> models...</blockquote><p>Suggestion: The usage of these models differ<s>s</s> from rest of the <code><s>imagery</s></code><s> and </s><code><s>text</s></code> models available in <s>offered by</s> <code>arcgis.learn</code> module in the sense that these models <s>doesn't</s> do not need to be trained on a given dataset before they can be used for inferencing. Therefore, these models do not <s>classes doesn't </s>have methods like <code>fit()</code>, <code>lr_find()</code> etc., which are required to train an <code>arcgis.learn</code> model.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Made the necessary changes</p>